File: bandlog-api/src/main/java/net/effize/bandlog/infrastructure/auth/exception/UserNotSignedUpException.java
package net.effize.bandlog.infrastructure.auth.exception;

import net.effize.bandlog.domain.common.exception.BandlogException;
import net.effize.bandlog.domain.common.exception.ErrorCode;

public class UserNotSignedUpException extends BandlogException {
    public UserNotSignedUpException() {
        super(ErrorCode.of("NOT_SIGNED_UP"), "Bandlog에 계정을 생성해 주세요.", "User is not signed up yet", 401);
    }
}

File: src/main/kotlin/net/effize/bandlog/auth/domain/exception/UserNotSignedUpException.kt
package net.effize.bandlog.auth.domain.exception

import net.effize.bandlog.common.exception.BandlogException
import net.effize.bandlog.common.exception.ErrorCode

class UserNotSignedUpException : BandlogException(
    errorCode = ErrorCode.of("NOT_SIGNED_UP"),
    userMessage = "Bandlog에 계정을 생성해 주세요.",
    logMessage = "User is not signed up yet",
    httpStatusCode = 401
)
